doc/whats_new/v0.21.rst

```diff
@@ -55,6 +56,13 @@ Support for Python 3.4 and below has been officially dropped.
   to set and that scales better, by :user:`Shane <espg>`,
   :user:`Adrin Jalali <adrinjalali>`, and :user:`Erich Schubert <kno10>`.
 
+:mod:`slkearn.decomposition`
+............................
+
+- |Fix| :class:`decomposition.SparseCoder` now passes `max_iter` to the
```

Comment:
This is now out of date. Please describe here what this change does, or at least update the PR description, as now it seems unclear, with the addition of `method_max_iter` and `transform_max_iter`